hawkeye-core/src/utils.rs

```diff
@@ -0,0 +1,27 @@
+use crate::config;
+use sentry::ClientInitGuard;
+use std::borrow::Cow;
+
+/// Helper for bootstrapping Sentry to capture panics and logs for context.
+pub fn maybe_bootstrap_sentry() -> Option<ClientInitGuard> {
+    if config::HAWKEYE_ENV.to_string() == "local".to_string() {
+        return None;
+    }
+    let mut log_builder = pretty_env_logger::formatted_builder();
+    log_builder.parse_filters("info");
+    let logger = sentry_log::SentryLogger::with_dest(log_builder.build());
+    log::set_boxed_logger(Box::new(logger)).unwrap();
+    // Log <= INFO as breadcrumbs. Anything higher is an "error" which generates a Sentry Issue.
+    log::set_max_level(log::LevelFilter::Info);
+
+    let sentry_client = sentry::init((
+        config::SENTRY_DSN.to_string(),
+        sentry::apply_defaults(sentry::ClientOptions {
+            release: sentry::release_name!(),
+            environment: Some(Cow::from(config::HAWKEYE_ENV.to_string())),
```

Comment:
The idea of them using `Cow` is to avoid creating unnecessary strings. Here I would suggest to use `.as_str()` instead of allocating a new `String` from `HAWKEYE_ENV` which is already a `String`.

```suggestion
            environment: Some(Cow::from(config::HAWKEYE_ENV.as_str())),
```